contributing/code/bugs.rst

```diff
@@ -32,9 +33,19 @@ If your problem definitely looks like a bug, report it using the official bug
 * Give as much detail as possible about your environment (OS, PHP version,
   Symfony version, enabled extensions, ...);
 
+* If you want to provide a stack trace you got on an html page, be sure to
```

Comment:
IMO sensitive data is the exception, not the rule, and a big drawback of using them that I did not mention is that you can't copy paste from them, making the work harder for people who want to check files mentioned in it. Also, these images can be resized and illegible. Also if not cropped properly, they might contain even more sensitive data, like what the desktop of the reporter looks like. Finally, even with a screenshot, you should redact sensitive information, so...